tests/wfc3/test_ir_01.py

```diff
@@ -0,0 +1,31 @@
+import subprocess
+
+from ..helpers import BaseWFC3
+
+
+class TestIR01(BaseWFC3):
+    """Tests for WFC3/IR."""
+    detector = 'ir'
+
+    def test_ir_01(self):
+        """Ported from ``calwf3_ir_01``."""
+        asn_file = 'iabg31030_asn.fits'
+
+        # Prepare input file.
+        self.get_input_file(asn_file)
+
+        # Run CALWF3
+        subprocess.call(['calwf3.e', asn_file, '-v'])
+
+        # Compare results
+        # Excluded ('iabg31031_spt.fits', 'iabg31031_spt_ref.fits')
```

Comment:
Note: There is a "truth" for SPT but local run does not generate SPT. Had an old note to ask @sosey but resolving this is also out of scope of this PR.